scripts/rss/index.js

```diff
@@ -0,0 +1,26 @@
+import fetch from 'node-fetch'
+
+import api from '../../config/api'
+import { getInitialState } from '../../src/getInitialState'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module '../../src/getInitialState'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->